moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -2315,19 +2315,19 @@ void RobotState::printTransforms(std::ostream& out) const
   const std::vector<const JointModel*>& jm = robot_model_->getJointModels();
   for (const JointModel* joint : jm)
   {
-    out << "  " << joint->getName();
+    out << ' ' << ' ' << joint->getName();
```

Comment:
No need to do this twice
```suggestion
    out << '  ' << joint->getName();
```

Reply:
:point_up: This applies to multiple changes in this code

Reply:
Is it possible to put two chars within the same ' '? It prints a numerical value when I do so

Reply:
@sjahr